src/main/java/core/service/impl/ReaderFromFileImp.java

```diff
@@ -0,0 +1,22 @@
+package core.service.impl;
+
+import core.service.ReaderFromFile;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ReaderFromFileImp implements ReaderFromFile {
+    @Override
+    public List<String> readFromFileData(String filePath) {
+        try(BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            return reader.lines().skip(1).collect(Collectors.toList());
```

Comment:
```suggestion
            return reader.lines().skip(OFFSET).collect(Collectors.toList());
```
let's make it more visible, maybe at some point the header will have different number of lines